connect/mirror/src/test/java/org/apache/kafka/connect/mirror/MirrorSourceTaskTest.java

```diff
@@ -81,15 +88,25 @@ public void testOffsetSync() {
         MirrorSourceTask.PartitionState partitionState = new MirrorSourceTask.PartitionState(50);
 
         assertTrue(partitionState.update(0, 100), "always emit offset sync on first update");
+        partitionState.reset();
```

Comment:
Is it worth adding coverage to this case (and possibly `testZeroOffsetSync`) for when we invoke `update` repeatedly without invoking `reset`?

Reply:
Increase the test coverage